--- cinelerra-1.2.1/cinelerra/Makefile.orig	2004-07-12 01:07:17.000000000 +0200
+++ cinelerra-1.2.1/cinelerra/Makefile	2004-08-22 20:29:36.344692752 +0200
@@ -1,18 +1,6 @@
 include ../global_config
 
-ALSA_DIR := ../alsa-lib-1.0.2
-AVC_DIR := ../libavc1394-0.4.1
-DV_DIR := ../quicktime/libdv-0.99
-FFMPEG_DIR := $(shell expr ../quicktime/ffmpeg* )
-LAME_DIR := ../quicktime/lame-3.93.1/include
-LIBRAW_DIR := ../libraw1394-0.9.0
-OGG_DIR := ../quicktime/libogg-1.0
-SNDFILE_DIR := $(shell expr ../libsndfile* )
-TIFF_DIR := ../tiff-v3.5.7
 TOOLAME_DIR := ../toolame-02l
-UUID_DIR := ../uuid
-VORBIS_DIR := ../quicktime/libvorbis-1.0
-EXR_DIR := ../OpenEXR-1.2.1/
 
 OBJS = \
 	$(OBJDIR)/aattachmentpoint.o \
@@ -285,24 +273,17 @@
 
 
 #ifeq ($(HAVE_AVIFILE), y)
-AVIFILE_DIR := $(shell expr ../avifile* )
+#AVIFILE_DIR := $(shell expr ../avifile* )
 #endif
 
 ifeq ($(HAVE_ALSA), y)
-ALSA_CFLAGS := -DHAVE_ALSA -I$(ALSA_DIR)/include
-ALSA_LIBS := $(ALSA_DIR)/$(OBJDIR)/libasound.a
+ALSA_CFLAGS := -DHAVE_ALSA
+ALSA_LIBS := -lasound
 endif
 
 STATICLIBS = \
-	$(AVC_DIR)/$(OBJDIR)/libavc.a \
-	../quicktime/$(OBJDIR)/libquicktime.a \
         $(ESOUNDLIBS) \
-	$(EXR_DIR)/$(OBJDIR)/libexr.a \
-	$(TIFF_DIR)/$(OBJDIR)/libtiff.a \
 	../guicast/$(OBJDIR)/libguicast.a \
-	../libmpeg3/$(OBJDIR)/libmpeg3.a \
-	$(SNDFILE_DIR)/$(OBJDIR)/libsndfile.a \
-	$(UUID_DIR)/$(OBJDIR)/libuuid.a \
 	../mpeg2enc/$(OBJDIR)/mpeg2enc.a \
 	$(TOOLAME_DIR)/$(OBJDIR)/toolame.a \
 	$(THEME_DATA)
@@ -323,7 +304,14 @@
 
 LIBS += \
 	$(ALSA_LIBS) \
-	$(LIBRAW_DIR)/$(OBJDIR)/libraw1394.a \
+	-lavc1394 -lrom1394 \
+	-lquicktime -lvorbisfile \
+	-lIlmImf -lImath -lHalf -lIex \
+	-ltiff \
+	-lmpeg3 \
+	-lsndfile \
+	-luuid \
+	-lraw1394 \
 	$(GLLIBS) \
 	-lX11 \
 	-lXext \
@@ -342,23 +330,17 @@
 
 CFLAGS += \
 	-I../guicast \
-	-I../quicktime \
-	-I$(DV_DIR) \
-	-I$(OGG_DIR)/include \
-	-I$(VORBIS_DIR)/include \
-	-I$(LIBRAW_DIR)/ \
-	-I$(TIFF_DIR)/libtiff \
-	-I$(UUID_DIR) \
-	-I$(DV_DIR) \
+	-I/usr/include/quicktime \
+	-I/usr/include/libdv \
+	-I/usr/include/libraw1394 \
+	-I/usr/include/uuid \
 	-I$(TOOLAME_DIR) \
-	-I$(LAME_DIR) \
-	-I$(AVC_DIR) \
-	-I../audiofile \
-	-I../libmpeg3 \
-	-I$(SNDFILE_DIR)/src \
+	-I/usr/include/lame \
+	-I/usr/include/libmpeg3 \
+	-I/usr/include/libavc1394 \
 	$(ESOUNDCFLAGS) \
 	$(ALSA_CFLAGS) \
-	-I$(FFMPEG_DIR)/libavcodec
+	-I/usr/include/ffmpeg
 
 
 # Speed up linking with this linking sequence
@@ -463,7 +445,7 @@
 
 $(FILEEXR): fileexr.C
 	@echo C++ $*.o
-	@$(CC) -c `cat $(OBJDIR)/c_flags` -I$(EXR_DIR)IlmImf -I$(EXR_DIR)Imath -I$(EXR_DIR)Iex -I$(EXR_DIR)Half $(subst $(OBJDIR)/,, $*.C) -o $*.o
+	@$(CC) -c `cat $(OBJDIR)/c_flags` -I/usr/include/OpenEXR $(subst $(OBJDIR)/,, $*.C) -o $*.o
 
 $(THEME_DATA):
 	cd $(OBJDIR) && \
--- cinelerra-1.1.8/cinelerra/jpegwrapper.h.orig	2003-10-14 09:54:01.000000000 +0200
+++ cinelerra-1.1.8/cinelerra/jpegwrapper.h	2003-11-12 22:28:31.358423944 +0100
@@ -7,7 +7,7 @@
 extern "C" {
 #endif
 
-#include "jpeg/jpeglib.h"
+#include <jpeglib.h>
 
 #ifdef __cplusplus
 }
--- cinelerra-1.1.8/mpeg2enc/Makefile.orig	2003-10-14 09:54:43.000000000 +0200
+++ cinelerra-1.1.8/mpeg2enc/Makefile	2003-11-12 22:23:56.387225920 +0100
@@ -1,6 +1,6 @@
 include ../global_config
 CC = gcc
-CFLAGS += -I../libmpeg3 -I../quicktime
+CFLAGS += -I/usr/include/libmpeg3 -I/usr/include/quicktime
 
 
 ifeq ($(OBJDIR), i686)
@@ -48,8 +48,8 @@
 	$(OBJDIR)/writepic.o
 
 LIBS = \
-	../../quicktime/$(OBJDIR)/libquicktime.a \
-	../../libmpeg3/$(OBJDIR)/libmpeg3.a \
+	-lquicktime \
+	-lmpeg3 \
 	-lpthread \
 	-lpng \
 	-lz \
--- cinelerra-1.1.8/mplexlo/Makefile.orig	2003-10-14 09:54:14.000000000 +0200
+++ cinelerra-1.1.8/mplexlo/Makefile	2003-11-12 22:06:01.235673800 +0100
@@ -2,12 +2,12 @@
 
 OBJS = $(OBJDIR)/mplex.o
 CC = gcc
-CFLAGS += -I../libmpeg3
+CFLAGS += -I/usr/include/libmpeg3
 
 OUTPUT := $(OBJDIR)/mplexlo
 
 $(OUTPUT): $(OBJS)
-	$(CC) -o $(OUTPUT) $(OBJS) ../libmpeg3/$(OBJDIR)/libmpeg3.a -lpthread -lm
+	$(CC) -o $(OUTPUT) $(OBJS) -lmpeg3 -lpthread -lm
 
 $(OBJS):
 	$(CC) -c $(CFLAGS) $(subst $(OBJDIR)/,, $*.c) -o $*.o
--- cinelerra-1.1.8/plugins/titler/Makefile.orig	2003-10-24 05:22:18.000000000 +0200
+++ cinelerra-1.1.8/plugins/titler/Makefile	2003-11-12 22:28:16.244721576 +0100
@@ -8,19 +8,18 @@
 
 
 OUTPUT_PLUGIN = ../$(OBJDIR)/titler.plugin
-FREETYPE_DIR = ../../freetype-2.1.4
-CFLAGS += -I$(FREETYPE_DIR)/include -I../../
+FREETYPE_DIR =
+CFLAGS += -I/usr/include/freetype2 -I../../
 
 
 $(OUTPUT_PLUGIN): $(OBJS)
 	$(LDLINKER) -o $(OUTPUT_PLUGIN) \
 		$(OBJS) \
-		$(FREETYPE_DIR)/$(OBJDIR)/libfreetype.a
-	strip $(OUTPUT_PLUGIN)
+		-lfreetype
 
 include ../../plugin_config
 
-inst:
+install:
 	cp $(OUTPUT_PLUGIN) /usr/lib/cinelerra
 	cp -a fonts /usr/lib/cinelerra
 
--- cinelerra-1.1.8/global_config.orig	2003-10-29 10:05:08.000000000 +0100
+++ cinelerra-1.1.8/global_config	2003-11-12 22:22:09.103535520 +0100
@@ -24,7 +24,7 @@
 
 ############################# FINAL DESTINATION FOR INSTALLATION BINARIES
 
-PREFIX = /usr/
+PREFIX = /usr
 LIBPREFIX = $(PREFIX)/lib
 
 ########################## THE COMPILER
@@ -50,8 +50,8 @@
 
 
 ifeq ($(HAVE_ESOUND), y)
-ESOUNDCFLAGS = -DHAVE_ESOUND -I../esound
-ESOUNDLIBS = ../esound/$(OBJDIR)/libesound.a
+ESOUNDCFLAGS = -DHAVE_ESOUND
+ESOUNDLIBS = -lesd
 endif
 
 
@@ -59,8 +59,8 @@
 CFLAGS += -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
 #CFLAGS += -I.      # Caused fileavi.C to fail
 
-FREETYPE_DIR = ../../freetype-2.1.4
-CFLAGS += -I$(FREETYPE_DIR)/include -I../../
+FREETYPE_DIR =
+CFLAGS += -I/usr/include/freeetype2
 
 ifeq ($(HAVE_OSS), y)
 CFLAGS += -DHAVE_OSS
--- cinelerra-1.1.8/plugin_config.orig	2003-10-14 09:53:58.000000000 +0200
+++ cinelerra-1.1.8/plugin_config	2003-11-12 22:22:42.189505688 +0100
@@ -1,6 +1,6 @@
 CFLAGS += -I../ -I../../cinelerra \
 	-I../../guicast \
-	-I../../quicktime \
+	-I/usr/include/quicktime \
 	-I../colors \
 	-fPIC
 
--- cinelerra-1.2.0/cinelerra/audioalsa.h.orig	2003-10-14 09:53:59.000000000 +0200
+++ cinelerra-1.2.0/cinelerra/audioalsa.h	2004-06-05 10:26:45.584744408 +0200
@@ -5,7 +5,7 @@
 #include "audiodevice.h"
 
 #ifdef HAVE_ALSA
-#include "asoundlib.h"
+#include <alsa/asoundlib.h>
 
 class AudioALSA : public AudioLowLevel
 {
--- cinelerra-1.2.0/cinelerra/avc1394control.h.orig	2004-04-09 09:22:12.000000000 +0200
+++ cinelerra-1.2.0/cinelerra/avc1394control.h	2004-06-05 10:27:24.377846960 +0200
@@ -2,11 +2,11 @@
 #define _AVC1394Control_H
 
 
-#include "librom1394/rom1394.h"
-#include "libavc1394/avc1394.h"
-#include "libavc1394/avc1394_vcr.h"
+#include <libavc1394/rom1394.h>
+#include <libavc1394/avc1394.h>
+#include <libavc1394/avc1394_vcr.h>
 #include "mutex.inc"
-#include "libraw1394/raw1394.h"
+#include <libraw1394/raw1394.h>
 #include <sys/types.h>
 #include <stdio.h>
 #include <errno.h>
--- cinelerra-1.2.1/cinelerra/device1394input.h.orig	2004-07-06 06:32:42.000000000 +0200
+++ cinelerra-1.2.1/cinelerra/device1394input.h	2004-08-22 21:26:29.328840296 +0200
@@ -7,7 +7,7 @@
 
 #include "condition.inc"
 #include "libdv.h"
-#include "libdv/dv1394.h"
+#include "dv1394.h"
 #include "mutex.inc"
 #include "libraw1394/raw1394.h"
 #include "thread.h"
--- cinelerra-1.2.1/cinelerra/device1394output.h.orig	2004-04-13 00:52:55.000000000 +0200
+++ cinelerra-1.2.1/cinelerra/device1394output.h	2004-08-22 21:23:11.293946176 +0200
@@ -8,7 +8,7 @@
 #include "audiodevice.inc"
 #include "condition.inc"
 #include "libdv.h"
-#include "libdv/dv1394.h"
+#include "dv1394.h"
 #include "ieee1394-ioctl.h"
 #include "mutex.inc"
 #include "thread.h"
