--- cinelerra-4/cinelerra/playback3d.h	2008-07-23 06:30:59.000000000 +0200
+++ cinelerra-4-new/cinelerra/playback3d.h	2008-10-11 19:43:52.841812309 +0200
@@ -246,7 +246,7 @@ public:
 // output - passed when rendering refresh frame.  If 0, the canvas is cleared.
 	void clear_output(Canvas *canvas, VFrame *output);
 
-	void do_fade(Canvas *canvas, VFrame *fade, float fade);
+	void do_fade(Canvas *canvas, VFrame *frame, float fade);
 
 	void do_mask(Canvas *canvas,
 		VFrame *output, 
