--- cinelerra/preferences.inc.orig	2012-09-22 13:38:33.839435794 +0100
+++ cinelerra/preferences.inc	2012-09-22 13:38:36.049436001 +0100
@@ -27,6 +27,12 @@
 #define BACKUP_PATH BCASTDIR "backup.xml"
 #define DEAMON_PORT 400
 
+#if defined(__x86_64__) || defined(__arch64__) || defined(__powerpc64__) || defined(__s390x__)
+#define PLUGIN_DIR "/usr/lib64/cinelerra"
+#else
+#define PLUGIN_DIR "/usr/lib/cinelerra"
+#endif
+
 class Preferences;
 class PlaybackConfig;
 class AudioOutConfig;
--- cinelerra/preferences.C.orig	2012-01-07 23:02:41.000000000 +0000
+++ cinelerra/preferences.C	2012-09-22 13:37:35.149430302 +0100
@@ -62,7 +62,7 @@
 
 
 
-	get_exe_path(plugin_dir);
+	sprintf(plugin_dir, PLUGIN_DIR);
 
 	sprintf(index_directory, BCASTDIR);
 	if(strlen(index_directory))
