--- cinelerra-1.1.5/plugins/defaulttheme/Makefile.orig	Sun Dec 22 11:37:27 2002
+++ cinelerra-1.1.5/plugins/defaulttheme/Makefile	Mon Mar 24 10:23:13 2003
@@ -6,7 +6,7 @@
 
 OUTPUT_THEME: $(OBJS)
 	$(LDLINKER) -o $(OUTPUT_THEME) $(OBJS)
-	strip $(OUTPUT_THEME)
+	$(STRIP) $(OUTPUT_THEME)
 	../../guicast/$(OBJDIR)/bootstrap $(OUTPUT_THEME) data/*.png
 
 
--- cinelerra-1.1.5/plugins/microtheme/Makefile.orig	Fri Jun 21 14:35:25 2002
+++ cinelerra-1.1.5/plugins/microtheme/Makefile	Mon Mar 24 10:23:34 2003
@@ -6,7 +6,7 @@
 
 OUTPUT_THEME: $(OBJS)
 	$(LDLINKER) -o $(OUTPUT_THEME) $(OBJS)
-	strip $(OUTPUT_THEME)
+	$(STRIP) $(OUTPUT_THEME)
 	../../cinelerra/$(OBJDIR)/bootstrap $(OUTPUT_THEME) data/*.png
 
 include ../../plugin_config
--- cinelerra-1.2.0/cinelerra/Makefile.orig	2004-06-05 01:31:27.950366992 +0200
+++ cinelerra-1.2.0/cinelerra/Makefile	2004-06-05 01:34:43.197684888 +0200
@@ -404,7 +404,7 @@
 	@echo LD $(OUTPUT)
 	@$(LINKER) `cat $(OBJDIR)/objs`
 	@echo STRIP $(OUTPUT)
-	@strip $(OUTPUT)
+	@$(STRIP) $(OUTPUT)
 	@echo BOOTSTRAP $(OUTPUT)
 	@../guicast/$(OBJDIR)/bootstrap $(OUTPUT) data/mode_*.png
 
