--- cinelerra-1.1.7/plugins/titler/title.C.orig	2003-10-14 23:51:02.000000000 +0200
+++ cinelerra-1.1.7/plugins/titler/title.C	2003-10-15 00:26:11.787038584 +0200
@@ -10,7 +10,7 @@
 #include "freetype/ftbbox.h"
 #include "freetype/ftglyph.h"
 #include "freetype/ftoutln.h"
-#include "freetype/ftstroker.h"
+#include FT_STROKER_H
 #include "picon_png.h"
 #include "plugincolors.h"
 #include "title.h"
