--- cinelerra-4/plugins/titler/title.C.orig	2008-08-07 05:33:08.000000000 +0200
+++ cinelerra-4/plugins/titler/title.C	2008-08-28 18:23:34.662127605 +0200
@@ -48,7 +48,7 @@
 #include <byteswap.h>
 #include <iconv.h>
 
-#define FONT_SEARCHPATH "/fonts"
+#define FONT_SEARCHPATH "/usr/share/fonts"
 
 REGISTER_PLUGIN(TitleMain)
 
@@ -982,8 +982,7 @@
 		fonts = new ArrayList<FontEntry*>;
 		char find_command[BCTEXTLEN];
 		sprintf(find_command, 
-			"find %s%s -name 'fonts.dir' -print -exec cat {} \\;", 
-			PluginClient::get_plugin_dir(),
+			"find %s -name 'fonts.dir' -print -exec cat {} \\;", 
 			FONT_SEARCHPATH);
 		FILE *in = popen(find_command, "r");
 		char current_dir[BCTEXTLEN];
