--- cinelerra-4/cinelerra/dvbtune.C.orig	2008-07-23 06:30:59.000000000 +0200
+++ cinelerra-4/cinelerra/dvbtune.C	2008-08-28 19:17:49.450113358 +0200
@@ -19,8 +19,8 @@
  * 
  */
 
-#include "../../guicast/bcwindowbase.inc"
-#include "../../guicast/clip.h"
+#include "../guicast/bcwindowbase.inc"
+#include "../guicast/clip.h"
 #include "condition.h"
 #include "devicedvbinput.inc"
 #include "dvbtune.h"
