--- cinelerra-4/cinelerra/Makefile.orig	2008-08-08 08:29:31.000000000 +0200
+++ cinelerra-4/cinelerra/Makefile	2008-08-28 17:53:28.910125929 +0200
@@ -1,33 +1,14 @@
 include ../global_config
 
-AVC_DIR := $(shell expr ../thirdparty/libavc1394* )
-DV_DIR := $(shell expr ../quicktime/thirdparty/libdv-*/ )
-FLAC_DIR := $(shell expr ../thirdparty/flac*/ )
-FFMPEG_DIR := $(shell expr ../quicktime/thirdparty/ffmpeg* )
-JPEG_DIR := $(shell expr ../quicktime/thirdparty/jpeg-mmx* )
-LAME_DIR := ../quicktime/thirdparty/lame-3.93.1/include
-LIBIEC_DIR := $(shell expr ../thirdparty/libiec61883* )
-LIBRAW_DIR := $(shell expr ../thirdparty/libraw1394* )
-OGG_DIR := $(shell expr ../quicktime/thirdparty/libogg* )
-SNDFILE_DIR := $(shell expr ../thirdparty/libsndfile* )
-THEORA_DIR := $(shell expr ../thirdparty/libtheora* )
-TIFF_DIR := ../thirdparty/tiff-v3.5.7
 TOOLAME_DIR := ../thirdparty/toolame-02l
-UUID_DIR := ../thirdparty/uuid
-VORBIS_DIR := $(shell expr ../quicktime/thirdparty/libvorbis* )
 
-AVC_LIB := $(AVC_DIR)/libavc1394/.libs/libavc1394.a $(AVC_DIR)/librom1394/.libs/librom1394.a
-RAW_LIB := $(LIBRAW_DIR)/src/.libs/libraw1394.a
-IEC_LIB := $(LIBIEC_DIR)/src/.libs/libiec61883.a
-SNDFILE_LIB := $(SNDFILE_DIR)/src/.libs/libsndfile.a
-THEORA_LIB := $(THEORA_DIR)/lib/.libs/libtheora.a
-EXR_LIBS :=  \
-	../thirdparty/lib/libIlmImf.a \
-	../thirdparty/lib/libIlmThread.a \
-	../thirdparty/lib/libImath.a \
-	../thirdparty/lib/libHalf.a \
-	../thirdparty/lib/libIex.a
-EXR_INCLUDES := -I../thirdparty/include/OpenEXR
+AVC_LIB := -lavc1394 -lrom1394
+RAW_LIB := -lraw1394
+IEC_LIB := -liec61883
+SNDFILE_LIB := -lsndfile
+THEORA_LIB := -ltheora
+EXR_LIBS :=  -lIlmImf -lIlmThread -lImath -lHalf -lIex
+EXR_INCLUDES := -I/usr/include/OpenEXR
 OBJS = \
 	$(OBJDIR)/aattachmentpoint.o \
 	$(OBJDIR)/aautomation.o \
@@ -331,34 +312,16 @@
 
 
 STATICLIBS = \
-	$(AVC_LIB) \
-	$(RAW_LIB) \
-	$(IEC_LIB) \
-	../quicktime/$(OBJDIR)/libquicktime.a \
-        $(ESOUNDLIBS) \
-	$(EXR_LIBS) \
-	$(FLAC_DIR)/src/libFLAC/.libs/libFLAC.a \
-	$(TIFF_DIR)/$(OBJDIR)/libtiff.a \
 	../guicast/$(OBJDIR)/libguicast.a \
-	../libmpeg3/$(OBJDIR)/libmpeg3.a \
-	$(SNDFILE_LIB) \
-	$(UUID_DIR)/$(OBJDIR)/libuuid.a \
 	../mpeg2enc/$(OBJDIR)/mpeg2enc.a \
 	$(TOOLAME_DIR)/$(OBJDIR)/toolame.a \
-	$(THEME_DATA) \
-	$(THEORA_LIB)
+	$(THEME_DATA)
 
 LIBS := \
 	-L./ \
 	-L../guicast
 
 
-ifeq ($(OBJDIR), x86_64)
-LIBS += -L/usr/X11R6/lib64
-else
-LIBS += -L/usr/X11R6/lib
-endif
-
 ifeq ($(HAVE_GL), y)
 LIBS += -lGL -lGLU
 endif
@@ -372,6 +334,19 @@
 
 
 LIBS += \
+	$(AVC_LIB) \
+	$(RAW_LIB) \
+	$(IEC_LIB) \
+	-lquicktime -lvorbisfile \
+	-lavformat \
+	$(ESOUNDLIBS) \
+	$(EXR_LIBS) \
+	-lFLAC \
+	-ltiff \
+	-lmpeg3 \
+	$(SNDFILE_LIB) \
+	-luuid \
+	$(THEORA_LIB) \
 	-lX11 \
 	-lXext \
 	-lXv \
@@ -390,26 +365,20 @@
 
 CFLAGS += \
 	-I../guicast \
-	-I../quicktime \
-	-I$(JPEG_DIR) \
-	-I$(DV_DIR) \
-	-I$(OGG_DIR)/include \
-	-I$(VORBIS_DIR)/include \
-	-I$(LIBRAW_DIR)/ \
-	-I$(AVC_DIR) \
-	-I$(LIBIEC_DIR)/src/ \
-	-I$(TIFF_DIR)/libtiff \
-	-I$(UUID_DIR) \
+	-I/usr/include/quicktime \
+	-I/usr/include/libdv \
+	-I/usr/include/libraw1394 \
+	-I/usr/include/libiec61883 \
+	-I/usr/include/uuid \
 	-I$(TOOLAME_DIR) \
-	-I$(LAME_DIR) \
-	-I../libmpeg3 \
-	-I$(SNDFILE_DIR)/src \
+	-I/usr/include/lame \
+	-I/usr/include/libmpeg3 \
 	$(ESOUNDCFLAGS) \
-	-I$(THEORA_DIR)/include
+	-I/usr/include/theora
 
 
 # Speed up linking with this linking sequence
-ifeq ($(OBJDIR), alpha)
+ifeq ($(OBJDIR), alpha-ccc)
 
 
 
@@ -431,10 +431,9 @@
 
 FFMPEG_CFLAGS := \
 	$(CFLAGS) \
-	-I$(FFMPEG_DIR)/libavcodec \
-	-I$(FFMPEG_DIR)/libavformat \
-	-I$(FFMPEG_DIR)/libavutil \
-	-I$(FFMPEG_DIR)
+	-I/usr/include/libavcodec \
+	-I/usr/include/libavformat \
+	-I/usr/include/libavutil
 
 
 $(shell echo $(CFLAGS) > $(OBJDIR)/c_flags)
@@ -519,7 +488,7 @@
 	$(CC) -c `cat $(OBJDIR)/c_flags` $(EXR_INCLUDES) $(subst $(OBJDIR)/,, $*.C) -o $*.o
 
 $(FILEFLAC): fileflac.C
-	$(CC) -c `cat $(OBJDIR)/c_flags` -I$(FLAC_DIR)/include $(subst $(OBJDIR)/,, $*.C) -o $*.o
+	$(CC) -c `cat $(OBJDIR)/c_flags` -I/usr/include/FLAC $(subst $(OBJDIR)/,, $*.C) -o $*.o
 
 $(FFMPEG_OBJS):
 	$(CC) -c `cat $(OBJDIR)/ffmpeg_cflags` $(subst $(OBJDIR)/,, $*.C) -o $*.o
--- cinelerra-1.1.8/mpeg2enc/Makefile.orig	2003-10-14 09:54:43.000000000 +0200
+++ cinelerra-1.1.8/mpeg2enc/Makefile	2003-11-12 22:23:56.387225920 +0100
@@ -1,6 +1,6 @@
 include ../global_config
 CC = gcc
-CFLAGS += -I../libmpeg3 -I../quicktime
+CFLAGS += -I/usr/include/libmpeg3 -I/usr/include/quicktime
 
 
 ifeq ($(OBJDIR), i686)
@@ -48,8 +48,8 @@
 	$(OBJDIR)/writepic.o
 
 LIBS = \
-	../../quicktime/$(OBJDIR)/libquicktime.a \
-	../../libmpeg3/$(OBJDIR)/libmpeg3.a \
+	-lquicktime \
+	-lmpeg3 \
 	-lpthread \
 	-lpng \
 	-lz \
--- cinelerra-1.1.8/mplexlo/Makefile.orig	2003-10-14 09:54:14.000000000 +0200
+++ cinelerra-1.1.8/mplexlo/Makefile	2003-11-12 22:06:01.235673800 +0100
@@ -2,12 +2,12 @@
 
 OBJS = $(OBJDIR)/mplex.o
 CC = gcc
-CFLAGS += -I../libmpeg3
+CFLAGS += -I/usr/include/libmpeg3
 
 OUTPUT := $(OBJDIR)/mplexlo
 
 $(OUTPUT): $(OBJS)
-	$(CC) -o $(OUTPUT) $(OBJS) ../libmpeg3/$(OBJDIR)/libmpeg3.a -lpthread -lm
+	$(CC) -o $(OUTPUT) $(OBJS) -lmpeg3 -lpthread -lm
 
 $(OBJS):
 	$(CC) -c $(CFLAGS) $(subst $(OBJDIR)/,, $*.c) -o $*.o
--- cinelerra-4/plugins/titler/Makefile.orig	2008-07-14 04:27:17.000000000 +0200
+++ cinelerra-4/plugins/titler/Makefile	2008-08-28 17:32:51.682116710 +0200
@@ -8,8 +8,8 @@
 
 
 OUTPUT = ../$(OBJDIR)/titler.plugin
-CFLAGS += -I$(FREETYPE_DIR)/include -I../../thirdparty/
-LFLAGS += $(FREETYPE_DIR)/$(OBJDIR)/libfreetype.a
+CFLAGS += -I/usr/include/freetype2
+LFLAGS += -lfreetype
 
 include ../../plugin_config
 
--- cinelerra-4/global_config.orig	2008-08-07 05:33:09.000000000 +0200
+++ cinelerra-4/global_config	2008-08-28 17:31:46.010117548 +0200
@@ -36,7 +36,7 @@
 
 ############################# FINAL DESTINATION FOR INSTALLATION BINARIES
 
-PREFIX = /usr/
+PREFIX = /usr
 LIBPREFIX = $(PREFIX)/lib
 
 
@@ -64,10 +64,10 @@
 
 
 ifeq ($(HAVE_ESOUND), y)
-ESOUND_DIR := ../thirdparty/esound
-AUDIOFILE_DIR := ../thirdparty/audiofile
-ESOUNDCFLAGS = -DHAVE_ESOUND -I$(ESOUND_DIR) -I$(AUDIOFILE_DIR)
-ESOUNDLIBS = $(ESOUND_DIR)/$(OBJDIR)/libesound.a
+AUDIOFILEDIR=
+ESOUNDDIR=
+ESOUNDCFLAGS = -DHAVE_ESOUND
+ESOUNDLIBS = -lesd
 endif
 
 
@@ -76,8 +76,8 @@
 #CFLAGS += -I.      # Caused fileavi.C to fail
 
 # the directory for plugin includes
-FREETYPE_DIR := $(shell expr ../../thirdparty/freetype-* )
-CFLAGS += -I$(FREETYPE_DIR)/include -I../../
+FREETYPE_DIR =
+CFLAGS += -I/usr/include/freetype2
 
 ifeq ($(HAVE_OSS), y)
 CFLAGS += -DHAVE_OSS
@@ -91,7 +91,7 @@
 CFLAGS += $(ALSA_CFLAGS)
 endif
 
-ifeq ($(OBJDIR), alpha)
+ifeq ($(OBJDIR), alpha-ccc)
 CXXREPOSITORY = cxx_repository/*.o
 endif
 
--- cinelerra-1.1.8/plugin_config.orig	2003-10-14 09:53:58.000000000 +0200
+++ cinelerra-1.1.8/plugin_config	2003-11-12 22:22:42.189505688 +0100
@@ -1,6 +1,6 @@
 CFLAGS += -I../ -I../../cinelerra \
 	-I../../guicast \
-	-I../../quicktime \
+	-I/usr/include/quicktime \
 	-I../colors \
 	-fPIC
 
--- cinelerra-2.0/guicast/Makefile.orig	2005-11-01 14:30:52.865839000 +0100
+++ cinelerra-2.0/guicast/Makefile	2005-11-01 14:41:01.161364152 +0100
@@ -2,9 +2,9 @@
 
 # Need the freetype include that matches the X server for Xft rendering
 #FREETYPE_DIR := ..//usr/X11R6/include/freetype2
-FREETYPE_DIR := $(shell expr ../freetype*/include )
+#FREETYPE_DIR := $(shell expr ../freetype*/include )
 
-CFLAGS += -I../quicktime -I$(FREETYPE_DIR)
+CFLAGS += -I/usr/include/quicktime -I/usr/include/freetype2
 
 # must only define here so applications using libpng don't see any of the XFT
 # includes at all.  They use a conflicting setjmp.
--- cinelerra-2.0/cinelerra/avc1394control.h.orig	2005-09-11 22:13:22.000000000 +0200
+++ cinelerra-2.0/cinelerra/avc1394control.h	2005-11-01 16:49:01.785731768 +0100
@@ -3,11 +3,11 @@
 
 
 #include "bcwindowbase.inc"
-#include "librom1394/rom1394.h"
-#include "libavc1394/avc1394.h"
-#include "libavc1394/avc1394_vcr.h"
+#include <libavc1394/rom1394.h>
+#include <libavc1394/avc1394.h>
+#include <libavc1394/avc1394_vcr.h>
 #include "mutex.inc"
-#include "libraw1394/raw1394.h"
+#include <libraw1394/raw1394.h>
 #include <sys/types.h>
 #include <stdio.h>
 #include <errno.h>
--- cinelerra-2.0/cinelerra/fileogg.h.orig	2005-09-07 23:25:21.000000000 +0200
+++ cinelerra-2.0/cinelerra/fileogg.h	2005-11-01 17:03:29.505818264 +0100
@@ -1,7 +1,7 @@
 #ifndef FILEOGG_H
 #define FILEOGG_H
 
-#include "../config.h"
+#include "../hvirtual_config.h"
 #include "filebase.h"
 #include "file.inc"
 
