--- cinelerra-1.2.1/plugins/defaulttheme/Makefile.orig	2004-05-11 18:14:51.000000000 +0200
+++ cinelerra-1.2.1/plugins/defaulttheme/Makefile	2004-08-22 20:38:29.715608048 +0200
@@ -9,7 +9,7 @@
 
 $(OUTPUT_THEME): $(OBJS) $(THEME_DATA)
 	$(LDLINKER) -o $(OUTPUT_THEME) $(OBJS) $(THEME_DATA)
-	strip $(OUTPUT_THEME)
+	$(STRIP) $(OUTPUT_THEME)
 
 
 
--- cinelerra-1.1.5/plugins/microtheme/Makefile.orig	Fri Jun 21 14:35:25 2002
+++ cinelerra-1.1.5/plugins/microtheme/Makefile	Mon Mar 24 10:23:34 2003
@@ -6,7 +6,7 @@
 
 OUTPUT_THEME: $(OBJS)
 	$(LDLINKER) -o $(OUTPUT_THEME) $(OBJS)
-	strip $(OUTPUT_THEME)
+	$(STRIP) $(OUTPUT_THEME)
 	../../cinelerra/$(OBJDIR)/bootstrap $(OUTPUT_THEME) data/*.png
 
 include ../../plugin_config
--- cinelerra-4/cinelerra/Makefile.orig	2008-08-28 18:02:51.486128443 +0200
+++ cinelerra-4/cinelerra/Makefile	2008-08-28 18:13:51.234129281 +0200
@@ -449,7 +449,7 @@
 # Also VFS only overrides the C library when dynamic linking is used.
 $(OUTPUT): $(OBJS) $(DCRAW) $(FILEEXR) $(FFMPEG_OBJS) $(FILEFLAC) $(THEME_DATA) $(SNDFILE_LIB)
 	$(LINKER) `cat $(OBJDIR)/objs`
-	strip $(OUTPUT)
+	$(STRIP) $(OUTPUT)
 
 $(OBJDIR)/test: 
 	$(CC) -shared -o $(OBJDIR)/test.so \
