Only in cinelerra-4.1.fix/bin: cinelerra
diff -ur cinelerra-4.1.orig/cinelerra/fileac3.C cinelerra-4.1.fix/cinelerra/fileac3.C
--- cinelerra-4.1.orig/cinelerra/fileac3.C	2010-08-25 13:18:55.974081519 +0200
+++ cinelerra-4.1.fix/cinelerra/fileac3.C	2010-08-25 13:36:19.960436542 +0200
@@ -19,6 +19,8 @@
  * 
  */
 
+#define __STDC_CONSTANT_MACROS
+
 #include "asset.h"
 
 extern "C" 
diff -ur cinelerra-4.1.orig/cinelerra/fileffmpeg.C cinelerra-4.1.fix/cinelerra/fileffmpeg.C
--- cinelerra-4.1.orig/cinelerra/fileffmpeg.C	2010-08-25 13:18:56.057207183 +0200
+++ cinelerra-4.1.fix/cinelerra/fileffmpeg.C	2010-08-25 13:36:03.847102952 +0200
@@ -19,6 +19,8 @@
  * 
  */
 
+#define __STDC_CONSTANT_MACROS
+
 #include "asset.h" 
 extern "C" 
 {
@@ -574,7 +576,7 @@
 			case PIX_FMT_YUV420P:
 				input_cmodel = BC_YUV420P;
 				break;
-			case PIX_FMT_YUV422:
+			case PIX_FMT_YUYV422:
 				input_cmodel = BC_YUV422;
 				break;
 			case PIX_FMT_YUV422P:
